node/src/main/scala/com/wavesplatform/database/package.scala

```diff
@@ -357,6 +357,33 @@ package object database {
     ndo.toByteArray
   }
 
+  def readBalanceInfo(bs: Array[Byte]): Map[Address, BalanceInfo] = {
+    val ndi        = newDataInput(bs)
+    val entryCount = ndi.readInt()
+
+    (0 until entryCount).map { _ =>
+      val addr          = ndi.readByteStr(Address.AddressLength)
```

Comment:
Please run explorer tool and check how much space is taken by addresses in these list. There's likely no need to store the whole address, address ID will do.